lms/envs/common.py

```diff
@@ -600,6 +600,17 @@
         'OPTIONS': {
             'name': 'tracking'
         }
+    },
+    'oli': {
+        'ENGINE': 'track.backends.oli.OLIAnalyticsBackend',
+        'OPTIONS': {
+            'oli_analytics_service_url': 'https://someurl/',
```

Comment:
Should this be `example.com` per IANA [1]?

[1] http://www.iana.org/domains/reserved